addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -29,7 +29,9 @@ def setUpClass(cls):
 
         # don't mess with existing leads, unlink those assigned to users used here to make tests
         # repeatable (archive is not sufficient because of lost leads)
-        cls.env['crm.lead'].with_context(active_test=False).search(['|', ('team_id', '=', False), ('user_id', 'in', cls.sales_teams.member_ids.ids)]).unlink()
+
+        with mute_logger('odoo.models.unlink'):
```

Comment:
Maybe we should just do that during tests in general? This log line exist specifically because "[deletions] leave no trace in database" (though arguably writes don't really either, a subsequent write to different fields of the same object will disappear them unless the information is specifically logged to the chatter).

For tests that just generates noise.

Reply:
I agree this could maybe be done for all tests. But I don't know if many tests will unlink data. We could check the logs.

Reply:
> I agree this could maybe be done for all tests. But I don't know if many tests will unlink data. We could check the logs.

Looking at the latest master, at_install reports "more than 1000", and the post_install batches report 651, 518, 347, 161, 49, and 335 (after some more thinking I figure the entire loading process could do without this).

I guess it might be valuable when trying to debug tours but that seems somewhat unlikely, I don't know that people would notice an unlink having gone missing.

An other possibility might be for the runbot to just run with that specific logger disabled :D

Reply:
> An other possibility might be for the runbot to just run with that specific logger disabled :D

Sneaky
I prefer to keep the runbot behaviour as closes as possible to the local one. We could maybe make this log conditionnal in models.py.

Reply:
> > An other possibility might be for the runbot to just run with that specific logger disabled :D
> 
> Sneaky I prefer to keep the runbot behaviour as closes as possible to the local one. We could maybe make this log conditionnal in models.py.

I mean sure, but manipulating `--log-handler` is not exactly a biggie. Especially since the runbot already has some special handling of a bespoke log level and stuff.